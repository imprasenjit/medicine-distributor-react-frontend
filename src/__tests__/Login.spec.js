import data from "../data"

test('Login Test', ()=>{
  expect(data).toMatchSnapshot();
})

