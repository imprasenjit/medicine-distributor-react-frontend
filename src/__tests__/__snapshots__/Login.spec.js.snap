// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Login Test 1`] = `
Array [
  Object {
    "name": "prasenjit",
    "phone": "1234",
  },
  Object {
    "name": "prasenjit2",
    "phone": "12345",
  },
]
`;
